doc/migrations/v0.46-v1.0.0.md

```diff
@@ -77,3 +78,7 @@ The following metrics were renamed:
 
 `libp2p_dialler_pending_dials` => `libp2p_dial_queue_pending_dials`
 `libp2p_dialler_in_progress_dials` => `libp2p_dial_queue_in_progress_dials`
+
+## Connection Manager
+
+We now only dial one peer at a time as this is a more efficient use of resources.  This means that the `maParallelDialsPerPeer` option has been removed.
```

Comment:
```suggestion
The observed behavior of dialing peers has been that given a list of supported addresses, if any one routable address would succeed then all would succeed and that if any routable address would fail then all would fail.

Consequently the previous dial behaviour of dialing all available addresses (up to a concurrency limit) and cancelling any in-flight dials when the first succeeds was a very inefficient use of resources.

Since `libp2p@0.46.10` we have only dialed one address at a time for each peer by setting the default value of the `ConnectionManager`'s `maxParallelDialsPerPeer` option to `1`.  As of `libp2p@1.0.0` this option has been removed.
```